updatehub/src/main.rs

```diff
@@ -89,6 +89,10 @@ struct ServerOptions {
     /// increase the verboseness level
     #[argh(option, short = 'v', from_str_fn(verbosity_level))]
     verbosity: Option<slog::Level>,
+
+    /// choose a conf file
+    #[argh(option, short = 'c')]
+    conf_file: Option<PathBuf>,
```

Comment:
The `argh` has support for default values. You could rewrite it like:

```suggestion
    #[argh(option, short = 'c', default = "PathBuf::from(\"/etc/updatehub.conf\")")]
    conf_file: PathBuf,
```